### DISCUSSION TITLE
Disable developer tools

### DISCUSSION THREAD

**Discussion by trufae** [ttytm/webview#18] [Q&A] [answered]
Is there a way to disable the "inspect element" from the context menu?

**Comment by ttytm** [comment, marked as answer]
Yes, this is possible by don't passing `(debug: true)` to the create function.
E.g.:
```v
webview.create()
```
Or maybe maybe adding a comptime conditon fits your needs. E.g.
```v
mut app := App{
	w: webview.create(
		debug: $if prod { false } $else { true }
	)
}
```

**Reply by trufae** [reply]
yeah thank you, this is exactly what i was looking for :) beautiful

**Reply by ttytm** [reply]
Nice good to hear that 🙂!
If you can hit the `Mark as answer` button to mark this question as answered I would appreciate it.
